go.mod

```diff
@@ -3,20 +3,20 @@ module github.com/authzed/authzed-go
 go 1.18
 
 require (
-	github.com/authzed/grpcutil v0.0.0-20210913124023-cad23ae5a9e8
+	github.com/authzed/grpcutil v0.0.0-20230509155820-7a6fedb71dbc
 	github.com/bufbuild/buf v1.19.0
-	github.com/envoyproxy/protoc-gen-validate v0.9.1
+	github.com/envoyproxy/protoc-gen-validate v1.0.1
 	github.com/golangci/golangci-lint v1.52.2
-	github.com/grpc-ecosystem/grpc-gateway/v2 v2.11.3
-	github.com/jzelinskie/stringz v0.0.0-20210414224931-d6a8ce844a70
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.2
+	github.com/jzelinskie/stringz v0.0.1
 	github.com/magefile/mage v1.14.0
-	github.com/planetscale/vtprotobuf v0.3.1-0.20220817155510-0ae748fd2007
-	github.com/stretchr/testify v1.8.2
+	github.com/planetscale/vtprotobuf v0.4.0
+	github.com/stretchr/testify v1.8.3
 	golang.org/x/sync v0.2.0
 	golang.org/x/vuln v0.1.0
-	google.golang.org/genproto v0.0.0-20230323212658-478b75c54725
+	google.golang.org/genproto v0.0.0-20230410155749-daa745c078e1
 	google.golang.org/grpc v1.54.0
```

Comment:
left gRPC intentionally at 1.54 because 1.55 breaks with the problem described in https://github.com/authzed/grpcutil/issues/48